packages/mobile/src/components/screen-wrapper/screen-wrapper.tsx

```diff
@@ -1,16 +1,24 @@
-import React, { FC } from 'react';
-import { StatusBar } from 'react-native';
+import React, { FC, ReactNode } from 'react';
+import { StatusBar, StatusBarProps, StyleProp, ViewStyle } from 'react-native';
 import {
-  NativeSafeAreaViewProps,
   SafeAreaView,
+  SafeAreaViewProps,
 } from 'react-native-safe-area-context';
 import { useCustomTheme } from '~/hooks/hooks';
 import { globalStyles } from '~/styles/styles';
 
-const ScreenWrapper: FC<NativeSafeAreaViewProps> = ({
+type Props = {
+  style?: StyleProp<ViewStyle>;
+  children: ReactNode;
+  statusBarProps?: StatusBarProps;
+  containerProps?: SafeAreaViewProps;
+};
+
+const ScreenWrapper: FC<Props> = ({
   style,
   children,
-  ...restProps
+  containerProps,
+  statusBarProps,
```

Comment:
Just apply new status bar component wherever you need it. No need to pass its props here